Dockerfile

```
@@ -62,6 +62,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 
 # Install setuptools, wheel and pip for Python3
 RUN wget https://bootstrap.pypa.io/get-pip.py -O - | python3
+RUN pip3 install meson
```

may be it is better to pin the version (as we do in requirement files) to prevent unpredictable behaviour.
```suggestion
RUN pip3 install meson==0.56.0
```